src/test/java/tests/EmptySidebarCartTest.java

```diff
@@ -0,0 +1,31 @@
+package tests;
+
+import fragments.HeaderFragment;
+import fragments.SidebarCartFragment;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+
+public class EmptySidebarCartTest extends BaseTest {
+
+    public static HeaderFragment headerFragment;
+    public static SidebarCartFragment sidebarCartFragment;
+    SoftAssert softAssert = new SoftAssert();
+
+    @BeforeMethod
+    public void beforeTest() {
+        headerFragment = new HeaderFragment(driver);
+        sidebarCartFragment = new SidebarCartFragment(driver);
+    }
+
+    @Test(description = "SPRIN-12 Empty Sidebar Cart Verification")
+    public void emptySidebarCart() {
+        sidebarCartFragment.clickCartIcon();
+
+        int itemsCount = sidebarCartFragment.getQuantityCount();
+        softAssert.assertEquals(itemsCount, 0, "The number of items in the cart sidebar is not zero");
```

Comment:
```suggestion
        softAssert.assertEquals(sidebarCartFragment.getQuantityCount(), 0, "The number of items in the cart sidebar is not zero");
```
U use itemsCount only one time, so you don't need to put it in variables